src/plugins/intel_npu/tests/unit/npuw/transpose_vt.cpp

```diff
@@ -0,0 +1,182 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <gtest/gtest.h>
+#include <common_test_utils/test_common.hpp>
+#include "openvino/op/ops.hpp"
+
+namespace npuw_utest{
+    using NodePtr = std::shared_ptr<ov::Node>;
+}
+
+//TODO: move to header maybe
+bool optimize_value_tensors(std::shared_ptr<ov::Model> model);
+
+enum class NetworkKind {
```

Comment:
We usually call it model type. Btw, those patterns work with wider range of models not only llama2 and llama-3, why they're called like this?

Reply:
I would say that other models just mimic patterns of either llama2 or llama3, that is why word "kind" came to place as a description of range of species. ModelType more narrow term as i see, but this also ok i have not preference.